requests/models.py

```diff
@@ -938,6 +938,8 @@ def raise_for_status(self):
 
         if http_error_msg:
             raise HTTPError(http_error_msg, response=self)
+        
```

Comment:
Please add a comment to the docstring, something like "alternatively, returns self on success"

Reply:
Add the comment in docstring